app/pages/nics/Class4DivingExemptPage.scala

```diff
@@ -16,15 +16,18 @@
 
 package pages.nics
 
+import controllers.journeys.nics.routes
+import models.NormalMode
 import models.common.{BusinessId, TaxYear}
 import models.database.UserAnswers
 import play.api.mvc.Call
 
 case object Class4DivingExemptPage extends NicsBasePage[List[BusinessId]] { // TODO type TBC, we may need a custom FormProvider
   override def toString: String = "class4DivingExempt"
 
-  override def nextPageInNormalMode(userAnswers: UserAnswers, businessId: BusinessId, taxYear: TaxYear): Call = cyaPage(taxYear, businessId)
+  override def nextPageInNormalMode(userAnswers: UserAnswers, businessId: BusinessId, taxYear: TaxYear): Call =
+    routes.Class4NonDivingExemptController.onPageLoad(taxYear, NormalMode)
```

Comment:
After exemption category page is updated, this will need a check to either NonDiving page or CYA page.
Same with the hasAllFurtherAnswers page below.